### PR TITLE
개수 구현

### PR DESCRIPTION
**Pull request by yhdev7935**
받아라, pull request. 이건 이다 commit.

### CODE REVIEW

File: web/src/components/itemList.js [hunk 1]
```diff
@@ -89,8 +94,8 @@ class ItemList extends Component {
                            const selected = [];
                            const currency = [];
                            for(let i = 0; i < data.getItems.length; i++) {
-                               selected.push(false);
-                               currency.push(false);
+                               selected.push(0);
+                               currency.push(0);
```
Comment (c0510gy) [inline web/src/components/itemList.js:98]: Why did you change the type of items that are inserted into currency from boolean to integer?

File: web/src/components/itemList.js [hunk 2]
```diff
@@ -67,11 +71,12 @@ class ItemList extends Component {
...
                     </Card.Text>
-                    <Button variant={
-                        this.selected[i] ? 'secondary' : 'primary'
-                    } id={i} onClick={this.itemClicked}>{
-                        this.selected[i] ? 'Remove from cart' : 'Add to cart'
-                    }</Button>
+                    
+                    <Button id={i} variant='primary' onClick={this.itemAdd}>Add this stuff</Button>
+                    <Card.Text>
+                        <NumberFormat value={this.selected[i]} displayType={'text'} thousandSeparator={true} prefix="담은 개수: " suffix="개" />
```
Comment (c0510gy) [inline web/src/components/itemList.js:77]: If you want to update the shape of the component, you should use state other than a general object.

File: web/src/components/itemList.js [hunk 3]
```diff
@@ -29,17 +29,21 @@ class ItemList extends Component {
...
+        this.props.addItems(this.state.items[idx].price);
+        this.selected[idx] += 1
+    }
+
+    itemRemove = event => {
+        const idx = event.target.getAttribute('id');
+        if(this.selected[idx] == 0)
+            alert('1개도 없어요!');
+        else
+        {
```
Comment (c0510gy) [inline web/src/components/itemList.js:43]: This is not following the code convention of this project.

**Comment by yhdev7935** [comment]
수정했음.

File: web/src/components/itemList.js [hunk 4]
```diff
@@ -24,25 +24,23 @@ class ItemList extends Component {
...
         };
-        this.selected = selected;
         this.rate = 0.001;
     }
 
     itemAdd = event => {
         const idx = event.target.getAttribute('id');
         this.props.addItems(this.state.items[idx].price);
-        this.selected[idx] += 1
+        this.state.selected[idx] += 1
```
Comment (c0510gy) [inline web/src/components/itemList.js:35]: This is the wrong way to update state.
The [React docs](https://reactjs.org/docs/react-component.html) says:

> Treat this.state as if it were immutable.

same problems on line 43
